repository.rosinstall

```diff
@@ -31,10 +31,14 @@
 - git:
     local-name: lma_kinematics_plugin
     uri: 'https://github.com/shadow-robot/lma_kinematics_plugin'
```

Comment:
Actually it seems that now this is part of the moveit repository.
See https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_kinematics/lma_kinematics_plugin

Can we test how it works without this source dependency (using the lma plugin in moveit).